src/software/ai/hl/stp/stp_tactic_assignment_test.cpp

```diff
@@ -1,677 +0,0 @@
-    // the goalie should always be assigned to the goalie assigned to the team
-    Team friendly_team(Duration::fromSeconds(0));
-    Robot robot_0(0, Point(-0.5, 0.2), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    // default is all capabilities, if not specified otherwise
-    Robot robot_1(1, Point(-0.5, -0.2), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    friendly_team.updateRobots({robot_0, robot_1});
-
-    friendly_team.assignGoalie(1);
-    world.updateFriendlyTeamState(friendly_team);
-
-    auto move_tactic_0 = std::make_shared<MoveTactic>();
-    auto move_tactic_1 = std::make_shared<MoveTactic>();
-
-    ConstPriorityTacticVector request = {{move_tactic_0, move_tactic_1}};
-
-    auto asst = stp.assignRobotsToTactics(request, world, true);
-
-    EXPECT_EQ(2, asst.size());
-
-    {
-        bool has_goalie = false;
-        bool has_move_0 = false;
-        for (const auto& [tactic, robot] : asst)
-        {
-            if (objectTypeName(*tactic) == TYPENAME(GoalieTactic))
-            {
-                has_goalie = true;
-                EXPECT_EQ(robot_1, robot);
-            }
-            if (tactic == move_tactic_0)
-            {
-                has_move_0 = true;
-                EXPECT_EQ(robot_0, robot);
-            }
-        }
-        EXPECT_TRUE(has_goalie);
-        EXPECT_TRUE(has_move_0);
-    }
-
-    request = {{move_tactic_0}, {move_tactic_1}};
-    asst    = stp.assignRobotsToTactics(request, world, true);
-
-    EXPECT_EQ(2, asst.size());
-
-    {
-        bool has_goalie = false;
-        bool has_move_0 = false;
-        for (const auto& [tactic, robot] : asst)
-        {
-            if (objectTypeName(*tactic) == TYPENAME(GoalieTactic))
-            {
-                has_goalie = true;
-                EXPECT_EQ(robot_1, robot);
-            }
-            if (tactic == move_tactic_0)
-            {
-                has_move_0 = true;
-                EXPECT_EQ(robot_0, robot);
-            }
-        }
-        EXPECT_TRUE(has_goalie);
-        EXPECT_TRUE(has_move_0);
-    }
-
-    request = {{move_tactic_1, move_tactic_0}};
-    asst    = stp.assignRobotsToTactics(request, world, true);
-
-    EXPECT_EQ(2, asst.size());
-
-    {
-        bool has_goalie = false;
-        bool has_move_1 = false;
-        for (const auto& [tactic, robot] : asst)
-        {
-            if (objectTypeName(*tactic) == TYPENAME(GoalieTactic))
-            {
-                has_goalie = true;
-                EXPECT_EQ(robot_1, robot);
-            }
-            if (tactic == move_tactic_1)
-            {
-                has_move_1 = true;
-                EXPECT_EQ(robot_0, robot);
-            }
-        }
-        EXPECT_TRUE(has_goalie);
-        EXPECT_TRUE(has_move_1);
-    }
-}
-
-TEST_F(STPTacticAssignmentTest, test_multi_tier_assignment_with_tiered_assignment)
```

Comment:
how come this test isn't included in the new test file?

Reply:
See above